core/src/config/redis.ts

```diff
@@ -35,6 +37,8 @@ function RealRedisConfig() {
     password,
     // If we have a REDIS_URL, but no database, assume we mean db=0
     db: parseInt(db || "0"),
+    // ssl options
+    tls: protocol === "rediss" ? { rejectUnauthorized: false } : undefined,
```

Comment:
is this `rediss` a standard of some sort? just looks like a typo.

Reply:
The extra `s` is for secure, lol

> As of Redis version 6, Hobby plans support both TLS and unencrypted connections, while production plans require TLS connections. For hobby plans you can find the unencrypted (plaintext) and TLS URIs in your app’s config vars REDIS_URL and REDIS_TLS_URL respectively. For Premium, Private, and Shield plans on version 6 your REDIS_URL config var will use the rediss: scheme, and you must enable TLS in your Redis client’s configuration in order to connect to a Redis 6 database.

https://devcenter.heroku.com/articles/heroku-redis